pixels/signal.py

```diff
@@ -16,7 +16,7 @@
 from pixels import ioutils, PixelsError
 
 
-def resample(array, from_hz, to_hz, padtype=None):
+def resample(array, from_hz, to_hz, poly=False, padtype=None):
```

Comment:
Can we keep the default as True? Then the existing behaviour for other uses of this function don't need to be changed.

Reply:
sure can do